### ISSUE TITLE
Update POA Request List Card Design

### ISSUE THREAD

**Issue by candilemoine** [department-of-veterans-affairs/va.gov-team#97453]
Ref: https://github.com/department-of-veterans-affairs/va.gov-team/issues/96866

### Description
We have updated designs for the card to flow horizontally. Let's update card structure.

### Implementation Details
- Status label
  - Pending
  - Accepted
  - Declined
- Name 
- City, State, Zip
- POA request date: [Date] (in X days)
  - Expiration for Pending requests
  - Accepted date for Accepted requests 
  - Declined date for Declined requests 
- If the date is 7 days or less, we should add the warning icon (see Figma)

Mockups/Designs
- [Card design](https://www.figma.com/design/LVCQBuW7a6nfVFNyhA4kv4/ARF---Form-21-22-Design-Explorations?node-id=1874-138808&node-type=frame&t=pgSxGYGo5bUHudF5-0)

### Acceptance Criteria/Conditions of Satisfaction

- [ ] a rep would be able to view a POA request info with the person's name, location, and when their POA request expires
- [ ] a rep would be able to see how many days left till the POA request expires
- [ ] a rep would see a warning sign if the POA request expired within X days or less
- [ ] if a POA request is new, it would show a Pending label
- [ ] if a POA request has been accepted by a rep, it would have an Accepted label and the accepted date
- [ ] if a POA request has been declined by a rep, it would have a Declined label and the declined date

#### Accessibility (A11y) Testing 
- Refer to existing [VA design system](https://design.va.gov/components/form/)
- Test with either [WAVE](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US&utm_source=ext_sidebar) or [AXE](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US&utm_source=ext_sidebar) extension
- As you manually test, look for the following:
   - color contrast errors
   - keyboard navigation
     - When navigating with a keyboard, content should flow from left to right
   - heading order 
     - Headings should be in order, so h1, then h2, h3. This is so the screen reader can let the user know the natural order of the elements in page. The WAVE tool has a headings tab that will display the order.
   - proper tag elements
- test with a screen reader 
  - Voiceover is built in with mac, use NVDA for windows

**Comment by jquispe-oddball** [comment]
Submitted pr https://github.com/department-of-veterans-affairs/vets-website/pull/33229

Screenshot of design with pending, declined, and accepted labels:
![Image](https://github.com/user-attachments/assets/596994c8-4d82-4845-9101-6a34f197dbb6)

**Comment by Kate-allbee** [comment]
@jquispe-oddball 
Hey there! Saw a couple of design changes to implement below: 
- Add space after each comma on  the address line. 
- Change yellow alert color to `vads-color-warning-dark`
- Card component was updated in the design system to change to height of the tag component within the card. [Link to thread here](https://dsva.slack.com/archives/C01DBGX4P45/p1732659281301679) Tag height should be 20px, design system team said the component update was pushed through to figma today. 
- If the POA request is showing the yellow icon, the request should have 7 or less days left before expiration. For this case we included a line after the date shown that counts down the days to expiration. I.e: "POA request expires on: November 25, 2024 (in 5 days)"
- ![Image](https://github.com/user-attachments/assets/6da9e20c-e365-46bb-b2af-293aa03109aa)

Thank you!
